src/components/Icons/TokenIcon.js

```diff
@@ -67,11 +68,9 @@ function TokenIcon({ url, size = 48, chain, style, imageStyle, chainIconStyle }:
       <Image source={source} style={[imageSizeStyle, imageStyle]} />
 
       {!!ChainIcon && (
-        <ChainIcon
-          width={chainIconSize}
-          height={chainIconSize}
-          style={[styles.chainIcon, chainIconSizeStyle, chainIconStyle]}
-        />
+        <View style={[styles.chainIconWrapper, chainIconSizeStyle, chainIconStyle]}>
```

Comment:
wdyt if we can use `chainIconWrapper` values as styled component? I mean styles are OK, but it's not widely used throughout our codebase, that's the only take

Reply:
Replaced